source/AI.cpp

```diff
@@ -1365,6 +1403,12 @@ bool AI::FollowOrders(Ship &ship, Command &command) const
 		else
 			command.SetTurn(TurnToward(ship, TargetAim(ship)));
 	}
+	else if(type == Orders::ATTACK && targetAsteroid)
+	{
+		MoveToAttack(ship, command, *targetAsteroid);
+		FireCommand fireCommand;
+		AutoFire(*it->first, fireCommand, *targetAsteroid);
```

Comment:
I assumed the ship would attack with this command but it does not... can anybody help me through this?

Reply:
This was implemented in the AI Step function.  I was looking at the wrong place originally.  I've cleaned up the messy code that was my old experiment.